zephyr/esp32/include/sdkconfig.h

```diff
@@ -9,6 +22,8 @@
 #define BTDM_CTRL_HCI_MODE_VHCI 1
 #define CONFIG_SPIRAM_ALLOW_BSS_SEG_EXTERNAL_MEMORY 1
 #define CONFIG_FREERTOS_UNICORE 1
-#define CONFIG_PARTITION_TABLE_OFFSET 0x8000
 #define CONFIG_IDF_FIRMWARE_CHIP_ID 0x0000
 #define CONFIG_SPIRAM_SIZE -1
+
+#define CONFIG_ESP32_XTAL_FREQ_40 1
```

Comment:
Why is this value different from the clock_frequency on cpu node in esp32.dts?

Reply:
forgot to clean it up, fixed